frontend/src/components/ProjectPicker.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div class="container">
+    <h1 class="title">Project Picker</h1>
+
+    <div class="new-projects">
+      <router-link :to="{ name: 'NewProject' }">
+        <button class="primary-button">Create New Project</button>
+      </router-link>
+    </div>
+
+    <!-- Display projects if available -->
+    <div class="projects-grid">
+      <div v-for="project in projects" :key="project.id" class="project-card">
+        <div class="delete-button" @click.stop="deleteProject(project.id)">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="white" width="18px" height="18px">
```

Comment:
Ich würde Icons immer entweder in eigene Komponenten auslagern oder direkt ne Icon library verwenden. (Wichtig: drauf achten das die treeshaking kann, sonst kann das Bundle schnell zu groß werden xD Machen aber eigentlich fast alle heute.)

Ich bin Fan von [Iconify](https://iconify.design/docs/icon-components/vue/), da hat man Zugriff auf extrem viele, gute Icons, ohne großen Aufwand. Gerade die `mdi:` icons nutze ich sehr gerne, da die ne gute Übersichtsseite haben (https://pictogrammers.com/library/mdi/)